src/rust.rs

```diff
@@ -853,9 +984,9 @@ impl<T: GCMethods + Copy> Heap<T> {
         self.ptr.get()
     }
 
-    pub fn handle(&self) -> Handle<T> {
+    pub fn handle(&self) -> RawHandle<T> {
```

Comment:
We should definitely return a non-raw Handle here, and return a MutableHandle from handle_mut that is not using the 'static lifetime.

Reply:
Are you sure that it's not using the `'static` lifetime now? I'm not so sure of it.

Reply:
I'm not even sure which code is being referred to any more. I believe that any code with a &self argument that returns a Handle will have the lifetime inferred to match the self argument, but it would be valuable to write that out explicitly.